pkg/config/integration_config.go

```diff
@@ -136,3 +138,25 @@ func NewIntegrationTestConfig(ctx context.Context, tb testing.TB) (*IntegrationT
 
 	return &cfg, db
 }
+
+// E2EConfig represents configurations to run server E2E tests.
+type E2EConfig struct {
+	DBName       string `env:"DB_NAME"`
+	APIServerURL string `env:"APISERVER_URL"`
+	AdminAPIURL  string `env:"ADMINAPI_URL"`
+	ProjectID    string `env:"PROJECT_ID"`
+	DBConfig     *database.Config
```

Comment:
This config also defines a `DB_NAME` (database.Name) value - how is that different than the one in this struct?

Either way, I think this should be:

```suggestion
	DBConfig     *database.Config `env:",prefix:E2E_"`
```

Reply:
Yup

Reply:
wow~ Awesome! This is a great feature!
@p3y1c2n please see my comment in `scripts.e2e-tests.sh`